contrib/kind.sh

```diff
@@ -293,6 +342,17 @@ install_j2_renderer() {
 }
 
 check_dependencies() {
+  if ! command_exists curl ; then
+    echo "Dependency not met: Command not found 'curl'"
+    exit 1
+  fi
+
+  if ! command_exists kubectl ; then
+    echo "Dependency not met: Command not found 'kubectl'"
+    echo "Downloading kubectl..'"
```

Comment:
Sorry for the super nit but I would remove 
```
echo "Dependency not met: Command not found 'kubectl'"
```
and replace
```
echo "Downloading kubectl..'"
```
with
```
echo "'kubectl' not found, installing"
```
for commonality with what is printed for j2 below.